common/eth2_network_config/Cargo.toml

```diff
@@ -7,7 +7,7 @@ edition = "2021"
 build = "build.rs"
 
 [build-dependencies]
-zip = "0.5.8"
+zip = "0.6"
```

Comment:
if we hold `zip` at 0.5 I think it avoids a couple of duplicate deps (like `aes`), but duplicating them is probably fine for now

Reply:
yup, thanks Michael updated it, also did the same on https://github.com/sigp/lighthouse/pull/4650

Reply:
Hmm looks like `zip 0.5` brought back a `time 0.1` dependency (looking at 281a26e6b2c78a05772d090a6c42cfc504fc1f0d), and we still have `zip 0.6` in some places too. Maybe we should just revert 281a26e6b2c78a05772d090a6c42cfc504fc1f0d and keep `zip 0.6` --  the other deps will hopefully get dragged into the present at some point.

Reply:
thanks for noticing Michael, yeah reverted. Meanwhile [`libp2p` has released an updated](https://github.com/libp2p/rust-libp2p/pull/4387#issuecomment-1692110164) versions with `webpki` dependency fixed. But we still depend on `webpki` via `tokio-tungstenite` (via `ethers-providers` and `warp`) and via `hyper-tls` I can address that after https://github.com/sigp/lighthouse/pull/4650

Reply:
addressed on https://github.com/sigp/lighthouse/pull/4650/commits/771e547c9c6d91dad1b2a47380ce32a7c392f014